monkey/infection_monkey/main.py

```diff
@@ -1,3 +1,7 @@
+# isort: off
+from serpentarium import PluginLoader  # noqa: F401
```

Comment:
You just need `import serpentarium`. It would be a good idea to include a comment as to why we're doing this here.